src/eclwatch/WUResult.tsx

```diff
@@ -73,7 +79,7 @@ function copyColumnTPL(col: number, dedup: boolean, fields: XSDXMLNode[], row: o
         set = row.map(r => valueTPL(r[name]));
     }
     return `
-SET OF ${typeTPL(fields[col].type)} ${name} := [${set.join(",")}];
+SET OF ${typeTPL(fields[col].type, fields[col].isSet)} ${name} := [${set.join(",")}];
```

Comment:
Could this result in "SET OF SET OF ___"?

Reply:
Nope - the same way we aren't getting a "STRING STRING"...

Reply:
@jbrundage - I "fixed" that, please re-review.